parsers/examples/example_parser.py

```diff
@@ -42,71 +51,12 @@ def fetch_production(
       ignored by the backend. If there is no data because the source may have
       changed or is not available, raise an ParserException.
 
-    A dictionary in the form:
-    {
-      'zoneKey': 'XX',
-      'datetime': '2017-01-01T00:00:00Z',
-      'production': {
-          'biomass': 0.0,
-          'coal': 0.0,
-          'gas': 0.0,
-          'hydro': 0.0,
-          'nuclear': null,
-          'oil': 0.0,
-          'solar': 0.0,
-          'wind': 0.0,
-          'geothermal': 0.0,
-          'unknown': 0.0
-      },
-      'storage': {
-          'hydro': -10.0,
-      },
-      'source': 'mysource.com'
-    }
-    or a list of dictionaries in the form:
-    [
-      {
-        'zoneKey': 'XX',
-        'datetime': '2017-01-01T00:00:00Z',
-        'production': {
-            'biomass': 0.0,
-            'coal': 0.0,
-            'gas': 0.0,
-            'hydro': 0.0,
-            'nuclear': null,
-            'oil': 0.0,
-            'solar': 0.0,
-            'wind': 0.0,
-            'geothermal': 0.0,
-            'unknown': 0.0
-        },
-        'storage': {
-            'hydro': -10.0,
-        },
-        'source': 'mysource.com'
-      },
-      {
-        'zoneKey': 'XX',
-        'datetime': '2017-01-01T01:00:00Z',
-        'production': {
-            'biomass': 0.0,
-            'coal': 0.0,
-            'gas': 0.0,
-            'hydro': 0.0,
-            'nuclear': null,
-            'oil': 0.0,
-            'solar': 0.0,
-            'wind': 0.0,
-            'geothermal': 0.0,
-            'unknown': 0.0
-        },
-        'storage': {
-            'hydro': -10.0,
-        },
-        'source': 'mysource.com'
-      },
-      ...
-    ]
+    A  ProductionBreakdownList should be returned containing all ProductionBreakdown
+    events. Each ProductionBreakdown event should contain a datetime, a zoneKey,
+    a ProductionMix, a source and optionally a StorageMix.
+    The ProductionMix should contain the production breakdown for each fuel type.
+    The StorageMix should contain the storage breakdown for each storage type.
```

Comment:
simple language suggestion to be more directive

```suggestion
   A  ProductionBreakdownList contains all ProductionBreakdown
    events. 
    Each ProductionBreakdown event contains a datetime, a zoneKey,
    a ProductionMix, a source and optionally a StorageMix.
    The ProductionMix contains the production breakdown for each fuel type.
    The StorageMix contains the storage breakdown for each storage type.
```